### DISCUSSION TITLE
k3d issue

### DISCUSSION THREAD

**Discussion by sudharshan-eGov** [egovernments/DIGIT-DevOps#824] [Q&A]
kubectl  get pods --all-namespaces  --kubeconfig /home/rentsher/.kube/fsm_admin_kubeconfig

exec plugin: invalid apiVersion "[client.authentication.k8s.io/v1alpha1](http://client.authentication.k8s.io/v1alpha1)"

**Comment by Karthik-egov** [comment]
try using the kubeconfig you get from the below commands
k3d kubeconfig get k3s-default > myk3dconfig
kubectl config use-context k3d-k3s-default --kubeconfig=myk3dconfig

Follow https://docs.digit.org/setup/quickstart/1.-infra-setup
